docs/source/tutorial/algorithm_cp_apr.ipynb

```diff
@@ -17,20 +17,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Set up a sample problem"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
+    "## Set up a sample problem\n",
     "We follow the general procedure outlined by E. C. Chi and T. G. Kolda, On Tensors, Sparsity, and Nonnegative Factorizations, arXiv:1112.2414 [math.NA], December 2011 (http://arxiv.org/abs/1112.2414)."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "vscode": {
+     "languageId": "python"
+    }
```

Comment:
Adding an addition `nbstripout` key to remove, as I had never seen these before (I don't use vscode for notebooks). The following will run on all tutorial files and fix all occurrences of the vscode metadata in the cells:

```
nbstripout docs/source/tutorial/*.ipynb --extra-keys "metadata.language_info metadata.vscode metadata.kernelspec cell.metadata.vscode"
```